helm-chart/renku-gateway/values.yaml

```diff
@@ -15,6 +15,11 @@ global:
   gitlab:
     urlPrefix: /gitlab
   gateway:
+    ## Optional list of allowed sources for Cross-Origin Resource Sharing.
+    ## https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+    # allowOrigin:
+    # - example.com
```

Comment:
Browsers include the schema in the origin header. I've found that when omitting the schema here, traefik will not include the necessary header in the response, so I'd update the example accordingly.

Reply:
I don't fully understand what you are asking. Are you just suggesting to add to the comment that "Providing no origins will result in Traefik not including the origin header altogheter"?

Reply:
Sorry for the clumsy wording. Since browsers include the schema in the origin header and traefik wants the origin header to exactly match one of the allowed origins, using examples like `https://example.com` and `http://foo.example.com` in the comment would be more useful.

Reply:
Yes, you're right, the example should include the protocol. I'll change it now!

Reply:
sorry, protocol, not schema 🤦